apps/meteor/tests/e2e/access-security-page.spec.ts

```diff
@@ -0,0 +1,86 @@
+import { Users } from './fixtures/userStates';
+import { AccountProfile } from './page-objects';
+import { getSettingValueById } from './utils/getSettingValueById';
+import { setSettingValueById } from './utils/setSettingValueById';
+import { test, expect, BaseTestAPI } from './utils/test';
+
+test.use({ storageState: Users.admin.state });
+
+test.describe.serial('access-security-page', () => {
+	let e2eEnabledDefaultValue: unknown;
+	let twofaEnabledDefaultValue: unknown;
+	let passwordChangeEnabledDefaultValue: unknown;
+	let poAccountProfile: AccountProfile;
+	let enableSettings: (api: BaseTestAPI, { passwordChange, twofa, e2e }: { passwordChange: boolean; twofa: boolean; e2e: boolean }) => Promise<void>;
+	let isFirstTestOver = false;
+
+	test.beforeAll(async ({ api }) => {
+		[passwordChangeEnabledDefaultValue, twofaEnabledDefaultValue, e2eEnabledDefaultValue] = await Promise.all([
+			getSettingValueById(api, 'Accounts_AllowPasswordChange'),
+			getSettingValueById(api, 'Accounts_TwoFactorAuthentication_Enabled'),
+			getSettingValueById(api, 'E2E_Enable'),
+		]);
+
+		enableSettings = async (api, { passwordChange, twofa, e2e }) => {
+			await Promise.all([
+				setSettingValueById(api, 'Accounts_AllowPasswordChange', passwordChange),
+				setSettingValueById(api, 'Accounts_TwoFactorAuthentication_Enabled', twofa),
+				setSettingValueById(api, 'E2E_Enable', e2e),
+			]);
+		}
+
+		await enableSettings(api, { passwordChange: false, twofa: false, e2e: false });
+	});
+
+	test.beforeEach(async ({ page }) => {
+		poAccountProfile = new AccountProfile(page);
+		await page.goto('/account/security');
+		await page.waitForLoadState('networkidle');
+	});
+
+	test.afterEach(async ({ api }) => {
+		if (!isFirstTestOver) {
```

Comment:
Hey, On updating the settings, the page needs to be refreshed in order the changes to be reflected in sidenav. I moved it to afterEach to save time. As sometimes the tests used to fail due to timeout. I will try to move it to the test.

Reply:
But from what the code is doing, we are executing this only once, after the first test case right?

In this case, I think it would be clearer to put it in a before inside the second block of the tests.